core-it-suite/src/test/resources/mng-7568-inactive-profile-deactivation/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.apache.maven.its.mng7568</groupId>
+    <artifactId>profile-selection</artifactId>
+    <version>1.0</version>
+
+    <profiles>
+        <profile>
+            <id>k-profile</id>
+        </profile>
+    </profiles>
+
+</project>
```

Comment:
Although it's not consistent over the whole project unfortunately, we prefer 2 spaces for xml files. Please adjust it here and in the settings.xml

Reply:
I took it from suggested `mng-7051-optional-profile-activation/pom.xml`, where it's set to 4. But will update this PR to 2.

Reply:
adjusted